samples/drivers/ipm/ipm_ivshmem/README.rst

```diff
@@ -0,0 +1,129 @@
+IPM over IVSHMEM Driver sample
+################################
+
+Prerequisites
+*************
+
+* QEMU needs to available.
+
+ivshmem-server needs to be available and running. The server is available in
+Zephyr SDK or pre-built in some distributions. Otherwise, it is available in
+QEMU source tree.
+
+ivshmem-client needs to be available as it is employed in this sample as an
+external application. The same conditions of ivshmem-server applies to the
+ivshmem-server, as it is also available via QEMU.
+
+Preparing IVSHMEM server
+************************
+#. The ivshmem-server utillity for QEMU can be found into the zephyr sdk
+   directory, in:
+   ``/path/to/your/zephyr-sdk/zephyr-<version>/sysroots/x86_64-pokysdk-linux/usr/xilinx/bin/``
+
+#. You may also find ivshmem-client utillity, it can be useful to debug if everything works
```

Comment:
s/utillity/utility/ ?

Reply:
Maybe s/to debug/to check/ since we debug things that are not working correctly?